src/styles/components/_navigation-panel.scss

```
@@ -2,7 +2,8 @@
 @import "input";
 
 .nav-panel {
+  padding: 0 100px 0 100px;
```

Better use shorthand padding expressing in this case
```suggestion
  padding: 0 100px;
```